.github/workflows/docker-web-dependencies.yml

```
@@ -66,7 +66,11 @@ jobs:
 
       - uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
 
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 69: could not find expected ':'

(syntax-check)